src/hotspot/share/cds/dumpAllocStats.hpp

```diff
@@ -146,6 +148,11 @@ class DumpAllocStats : public StackObj {
     _num_method_cp_entries_reverted += reverted ? 1 : 0;
   }
 
+  void record_dynamic_proxy_class() {
```

Comment:
This is not called. This code seems not related.

Reply:
True. @iklam, this came with a change you wanted me to take. Ok to cut this out?

Reply:
Yes, this part is not needed. AOT dynamic proxy classes are only supported in the Leyden repo.